src/assets/logos/index.js

```diff
@@ -4,20 +4,20 @@
 
 import styled from 'styled-components';
 
-import BCMLogoPNG from './research-logos/Baylor-College-of-Medicine.png';
-import BYULogoPNG from './research-logos/BYU.png';
+import BCMLogoPNG from './research-logos/baylor-college-of-medicine.svg';
+import BYULogoPNG from './research-logos/BYU.svg';
 import CFRLogoPNG from './42cfrp2.png';
 import CJISLogoPNG from './cjis.png';
 import FERPALogoPNG from './ferpa.png';
-import GeorgetownLogoPNG from './research-logos/Georgetown-University.png';
+import GeorgetownLogoPNG from './research-logos/Georgetown.png';
```

Comment:
how did this png get 7x bigger?

Reply:
ah it's a different png actually—different logo.